app/Http/Controllers/User/BookController.php

```diff
@@ -152,6 +152,8 @@ public function store(BookRequest $request)
                     $this->notification->store($info);
                 }
             }
+            \Cache::put('latestBook', $this->book->setCache(['created_at','desc']), 1440);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/BookController.php</br>**Line**: 155</br>Expected 1 space between comma and "'desc'"; 0 found